src/client/index.base.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import debug from 'debug';
+import CookieSerializer from 'CookieSerializer';
+import Router from 'react-router';
+import makeLoadLocale from './loadLocale';
+import { query, runCommand } from 'avenger/lib/graph';
+import run from 'buildo-state/lib/run';
+import { QueriesContextTypes, CommandsContextTypes } from 'container';
+import { makeOnBrowserChange, makeSyncToBrowser } from 'state-react-router';
+
+import 'assets';
+
+const log = debug('app:client');
+
+export default function({
+  config,
+  routes, // required
+  loadLocaleMessages = () => new Promise(resolve => {
+    require(['locales/en'], resolve);
+  }),
```

Comment:
I don't think this default is useful as of now, could you try removing it and test if everything is still 👍 ?